tests/index.spec.js

```diff
@@ -223,9 +223,10 @@ describe('Testing generic actions', () => {
       expect(e.list[0].message).to.equal('fioAddress must have a length between 3 and 64.')
     }
     try {
-      FIOSDK.isFioDomainValid('$%FG%')
+//      FIOSDK.isFioDomainValid('$%FG%')
     } catch (e) {
-      expect(e.list[0].message).to.equal('fioDomain must match /^[a-z0-9\\-]+$/i.')
+//      console.log(e)
+//      expect(e.list[0].message).to.equal('fioDomain must match /^[a-z0-9\\-]+$/i.')
```

Comment:
@ericbutz Could you please pull latest update for this validation check?

Reply:
Uncommented these in latest. Marking as done.